packages/runtime/src/next/api-handler/handlers/proxy-draft-mode.test.ts

```diff
@@ -0,0 +1,119 @@
+import { NextRequest } from 'next/server'
+import { http, HttpResponse } from 'msw'
+import { server } from '../../../mocks/server'
+import proxyDraftMode from './proxy-draft-mode'
+import { env } from 'process'
+
+jest.mock('next/headers', () => ({
+  draftMode: jest.fn().mockReturnValue({ enable: jest.fn(), disable: jest.fn() }),
+  cookies: jest.fn().mockReturnValue({ get: jest.fn() }),
+}))
+
+const mockApiKey = 'test-api-key'
+  
+beforeAll(() => {
+  server.listen()
+})
+
+afterAll(() => {
+  console.log('afterAll')
+  server.close()
+})
+
+afterEach(() => {
+  console.log('afterEach')
+  server.resetHandlers()
+  jest.clearAllMocks()
+})
+
+describe('ForceHTTP proxyDraftMode URL handling', () => {
+  beforeAll(() => {
+    env.FORCE_HTTP = 'true'
+  })
+
+  afterAll(() => {
+    env.FORCE_HTTP = 'false'
```

Comment:
This seems to be setting the variable to a _truthy_ value, which doesn't unset it. Our implementation checks for the env var being `null`